src/components/activities/ActivityCard.jsx

```diff
@@ -7,16 +7,28 @@ import PropType from 'prop-types';
  * @extends React.Component
 */
 class ActivityCard extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {};
-    this.statuses = ['pending', 'expired', 'approved', 'default'];
-  }
-
+  /**
+  * @name propTypes
+  * @type {PropType}
+  * @param {Object} propTypes - React PropTypes
+  * @property {String} category - The type of an Activity
+  * @property {String} date - The date on which a fellow participated in an activity
+  * @property {String} description - The description of the activity
+  * @property {String} points - The points the activity is worth
+  * @property {String} status - The current status of the activity
+  */
+  static propTypes = {
+    category: PropType.string.isRequired,
+    date: PropType.string.isRequired,
+    description: PropType.string.isRequired,
+    points: PropType.string.isRequired,
+    status: PropType.string.isRequired,
+  };
+  statuses = ['pending', 'expired', 'approved', 'default'];
   /**
    * @summary Renders the status indicator on the ActivityCard
    */
-  renderStatus() {
+  renderStatus= () => {
```

Comment:
Add space before equal sign.

Reply:
@john555 @joykare @Farhanisto Addressing style rules on PR reviews is a good practise, on the other hand though it makes PR reviews noisy and distracting from the global objective i.e to make sure code is correct, secure and flexible. I have an opinion, that using linters and CI to make sure such minor details don't make it to this level will be much better. I'm suggesting we adopt this in our work flow, any thoughts on this @team?